agent/acs/handler/acs_handler.go

```diff
@@ -257,14 +221,23 @@ func (acsSession *session) Start() error {
 // startSessionOnce creates a session with ACS and handles requests using the passed
 // in arguments
 func (acsSession *session) startSessionOnce() error {
+	minAgentCfg := &wsclient.WSClientMinAgentConfig{
+		AcceptInsecureCert: acsSession.agentConfig.AcceptInsecureCert,
+		AWSRegion:          acsSession.agentConfig.AWSRegion,
+	}
+
 	acsEndpoint, err := acsSession.ecsClient.DiscoverPollEndpoint(acsSession.containerInstanceARN)
 	if err != nil {
 		seelog.Errorf("acs: unable to discover poll endpoint, err: %v", err)
 		return err
 	}
 
-	url := acsWsURL(acsEndpoint, acsSession.agentConfig.Cluster, acsSession.containerInstanceARN, acsSession.taskEngine, acsSession.resources)
-	client := acsSession.resources.createACSClient(url, acsSession.agentConfig)
+	url := acsSession.acsWsURL(acsEndpoint, acsSession.agentConfig.Cluster, acsSession.containerInstanceARN)
```

Comment:
```
url := acsSession.acsWsURL(acsEndpoint, acsSession.agentConfig.Cluster, acsSession.containerInstanceARN)
```
do we need to pass acsSession.agentConfig.Cluster, acsSession.containerInstanceARN since we already have access to acsSession object?

Reply:
We do not - thanks for calling that out! I will update this.